test/unit/StakingPool/depositTo.js

```diff
@@ -480,7 +480,7 @@ describe('depositTo', function () {
 
     const { firstActiveTrancheId, maxTranche } = await getTranches(daysToSeconds(0), DEFAULT_GRACE_PERIOD);
 
-    const tranches = Array(maxTranche - firstActiveTrancheId + 1)
```

Comment:
let's undo this, there is a fix on the parent branch for the underlying issue already.